tests/sparse_weights_test.py

```diff
@@ -100,6 +100,78 @@ def test_rezero_after_forward_2d(self):
                 expected = [round(input_size * (1.0 - sparsity))] * out_channels
                 self.assertSequenceEqual(counts.numpy().tolist(), expected)
 
+    def test_fully_dense_linear(self):
+
+        lin = torch.nn.Linear(10, 10)
+        sw = SparseWeights(lin, sparsity=0)
```

Comment:
Wrapping a dense layer with `sparsity=0` makes no sense. It wont change anything on the original layer